lib/gui/app/components/image-selector/image-selector.jsx

```diff
@@ -0,0 +1,152 @@
+/*
+ * Copyright 2016 resin.io
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+ 'use strict'
+
+const React = require('react')
+const ReactDOM = require('react-dom')
+const propTypes = require('prop-types')
+const Color = require('color')
+
+const middleEllipsis = require('./../../utils/middle-ellipsis')
+
+const styled = require('styled-components').default
+const {Button, Box, Txt, Provider} = require('rendition')
+
+const shared = require('/./../../../../../lib/shared/units')
+const { colors, consts } = require('./../../theme')
+
+const StyledButton = styled(Button)`
+  width: 100%;
+  max-width: ${consts.btnMinWidth};
+  margin: auto;
+
+  overflow: hidden;
+`
+
+const StyledFooter = styled(Txt.p).attrs({
+  className: 'step-footer'
+})``
```

Comment:
The trailing `` is not needed here, or on the components below. If the component isn't being styled using styled-components, you're better off simply adding a class to the component you're wrapping, so that `<StyledFooter/>` becomes `<Txt.p className="step-footer" />`.

Reply:
I wanted to add the styled wrapper class for soon-to-be-made changes and to have template ready.